libs/core/src/components/button/fab/fab.ts

```diff
@@ -35,15 +35,29 @@ export class GdsFab extends GdsButton {
   })
   transform?: string
 
-  /**
-   * The inset postion of the FAB.
-   *
-   * Accepts any valid CSS inset values.
-   */
   @styleExpressionProperty({
     valueTemplate: (v) => v,
+    selector: '[part="_button"]',
```

Comment:
I was actually experimenting with this one, and reason was that you have to know the order it goes when using inset  and working with it was more efficient knowing exactly where it is going to end up. But no other reason than that only usability.

Reply:
I see! I think we should stick to short-hand properties for these style expressions wherever possible though, since it reduces the amount of code and number of properties by quite a bit, and since the order and the way it works is the same as CSS, it shouldn't make it too difficult to use.

Reply:
Yeah I agree on that and if you need to change on different breakpoints I guess much more efficient with inset!

Reply:
Switched back to using inset and updated the stories also!